### PR TITLE
WRN-3818: Migrate ExpandableInput, ExpandableItem, ExpandableList and ExpandablePicker unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist
* [X] I have read and understand the contribution guide
* [ ] A CHANGELOG entry is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Migrated tests from Enzyme to Testing Library.

### Resolution
Changed test structure to work with Testing Library.

### Links
WRN-3818

### Comments

### CODE REVIEW

File: ExpandableInput/tests/ExpandableInput-specs.js [hunk 1]
```diff
@@ -1,109 +1,82 @@
...
+			const inputField = screen.getByTestId('inputField').firstElementChild;
+
+			const expectedValue = 'Item value' + inputHint;
+			const expectedAttribute = 'aria-label';
+
+			expect(inputField).toHaveAttribute(expectedAttribute, expectedValue);
+		});
+
+		test('should use title, noneText, and input hint when value is not set', () => {
+			render(<ExpandableInputBase data-testid="inputField" title="Item" noneText="noneText" />);
```
Comment (stanca-pop-lgp) [inline ExpandableInput/tests/ExpandableInput-specs.js]: Please move noneText prop before title.

File: ExpandableInput/tests/ExpandableInput-specs.js [hunk 2]
```diff
@@ -1,109 +1,82 @@
...
-					<ExpandableInputBase title="Item" type="password" value="1" />
-				);
-
-				const expected = 'Item 1 character' + inputHint;
-				const actual = subject.prop('aria-label');
-
-				expect(actual).toBe(expected);
-			}
-		);
+		test('should use title, single character count, and input hint when type is `password` and value length is 1', () => {
```
Comment (stanca-pop-lgp) [inline ExpandableInput/tests/ExpandableInput-specs.js]: This line is too long.

File: ExpandableInput/tests/ExpandableInput-specs.js [hunk 3]
```diff
@@ -1,109 +1,82 @@
...
 		});
 
 		test('should use noneText when value is not set', () => {
-			const subject = shallow(
-				<ExpandableInputBase title="Item" noneText="noneText" />
-			);
-
-			const expected = 'noneText';
-			const actual = subject.prop('label');
+			render(<ExpandableInputBase title="Item" noneText="noneText" />);
```
Comment (stanca-pop-lgp) [inline ExpandableInput/tests/ExpandableInput-specs.js]: Please sort props. Below as well.

File: ExpandableItem/tests/ExpandableItem-specs.js [hunk 4]
```diff
@@ -38,55 +43,52 @@ describe('ExpandableItem', () => {
 
 		describe('handlers', () => {
 			test('should call onClose when there is a prop onClose', () => {
-				const children = ['option1', 'option2', 'option3'];
 				const handleClose = jest.fn();
-
-				const expandableItem = mount(
+				render(
 					<ExpandableItemBase onClose={handleClose} title="Item" noneText="hello" open>
```
Comment (stanca-pop-lgp) [inline ExpandableItem/tests/ExpandableItem-specs.js]: Please sort props alphabetically. Below as well.

File: ExpandableList/tests/ExpandableList-specs.js [hunk 5]
```diff
@@ -1,86 +1,110 @@
...
 import {ExpandableList, ExpandableListBase} from '../ExpandableList';
 
 describe('ExpandableList', () => {
+	const children = ['option1', 'option2', 'option3'];
+
 	describe('#aria-multiselectable', () => {
 		test('should be true when select is multiple', () => {
-			const expandableList = mount(
+			render(
 				<ExpandableListBase title="Item" select="multiple">
```
Comment (stanca-pop-lgp) [inline ExpandableList/tests/ExpandableList-specs.js]: Please sort props for all tests.

File: ExpandablePicker/tests/ExpandablePicker-specs.js [hunk 6]
```diff
@@ -43,118 +43,104 @@ describe('ExpandablePicker Specs', () => {
...
+		expect(joinedPicker).toHaveClass(expected);
 	});
 
 	test('should set "data-webos-voice-disabled" to decrement button when voice control is disabled', () => {
-		const children = ['option1', 'option2', 'option3'];
-
-		const expandablePicker = mount(
-			<ExpandablePicker data-webos-voice-disabled title="Options" open>
+		render(
+			<ExpandablePicker decrementAriaLabel="decrementer" data-webos-voice-disabled title="Options" open>
```
Comment (stanca-pop-lgp) [inline ExpandablePicker/tests/ExpandablePicker-specs.js]: Please sort props for this test and below.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Pull request merged**